pkg/polaris/ctl_polaris.go

```diff
@@ -118,9 +119,9 @@ func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Comma
 	cmd.Flags().StringVar(&ctl.EnvironmentDNS, "environment-dns", ctl.EnvironmentDNS, "Environment DNS")
 	cmd.Flags().StringVar(&ctl.ImagePullSecrets, "pull-secret", ctl.ImagePullSecrets, "Pull secret")
 	cmd.Flags().StringVar(&ctl.StorageClass, "storage-class", ctl.StorageClass, "Set the storage class to use for all the PVC")
-	cmd.Flags().BoolVar(&ctl.EnableReporting, "enable-reporting", false, "Send this flag if you wish to enable ReportingPlatform")
+	cmd.Flags().BoolVar(&ctl.EnableReporting, "enable-reporting", false, fmt.Sprintf("Send this flag if you wish to enable ReportingPlatform. Default: %s", strconv.FormatBool(GetPolarisDefault().EnableReporting)))
 	cmd.Flags().StringVar(&ctl.GCPServiceAccount, "gcp-service-account-path", ctl.GCPServiceAccount, "Google Cloud Service account")
-	cmd.Flags().StringVar(&ctl.IngressClass, "ingress-class", ctl.IngressClass, "The name of the ingress class to use. Default is nginx.")
+	cmd.Flags().StringVar(&ctl.IngressClass, "ingress-class", ctl.IngressClass, fmt.Sprintf("The name of the ingress class to use. Default: %s", GetPolarisDefault().IngressClass))
```

Comment:
All the string var, in cobra, the third parameter is already meant for `default` values, so you could just put it there, and change the fourth parameter to be string as is.

Reply:
This, for example https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.11.x/pkg/blackduck/ctl_blackduck.go#L162

Reply:
Updated